Google.Api.Generator/Generation/ServiceDetails.cs

```diff
@@ -117,5 +122,8 @@ public ServiceDetails(ProtoCatalog catalog, string ns, ServiceDescriptor desc, S
 
         /// <summary>Grpc Service-Config Method configs, includes both service-level and method-level.</summary>
         public IReadOnlyDictionary<string, MethodConfig> MethodGrpcConfigsByName { get; }
+
+        /// <summary>Name of the proto package for this service</summary>
+        public string Package { get; }
```

Comment:
nit: Maybe reanme to ProtoPackage so it's inmediately clear?

Reply:
👍